### DISCUSSION TITLE
404 error only with specific links

### DISCUSSION THREAD

**Discussion by sweettea91** [community/community#48406] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hello github community,

I published my portfolio on github pages and all seemed to be going well but then I noticed that there are specific links that get a 404 file not found error. I'm trying to link my portfolio-details pages back to my index.html

![Capturegit](https://user-images.githubusercontent.com/117620564/220984586-0de527a2-d944-4faf-9007-00218a0a8bcb.JPG)

It's the same issue for all my portfolio-details pages. and I don't understand why the link works in one area of my page and not the other. Here is a link to my repository and published page to get a better look

https://github.com/sweettea91/portfolio.git

https://sweettea91.github.io/portfolio/

**Comment by radar-blip** [comment]
Everything seems to be working fine for me. No 404

**Reply by sweettea91** [reply]
thank you, it was not working last night so I'm guessing that my changes I made last night took a little bit to update.

**Reply by radar-blip** [reply]
You're welcome.
I just checked again, clicked on all links multiple times, and everything seems to be working as it should.

**Comment by dqwira** [comment]
It looks like all the links in the navigation menu are pointing to the same index.html file, which could be why you're getting a 404 error when trying to access specific pages. Make sure to update the href attribute for each portfolio-details page

**Comment by sweettea91** [comment, marked as answer]
thanks! I think it took a little bit to update. 
for Portolio-details-1.html, I updated it last night to all point to the same index.html file.
before it would point to index.html #about, index.html #resume, which worked on my local file
I just updated the other portfolio-details pages and removed the #directory for each index.html link.
hopefully this works, but does anyone know how long it takes to update on the published page after committing the changes?

**Reply by dqwira** [reply]
Usually, you should be able to see the changes in a few minutes. To check if the changes have been applied, you can try clearing your browser cache or opening the page in a private browsing window to see if the changes have been applied.
